src/main/java/uk/ac/ebi/ena/webin/cli/manifest/processor/metadata/SampleProcessor.java

```diff
@@ -47,29 +55,98 @@ public Callback<Sample> getCallback() {
-
+      if (isJsonSample(sampleValue)) {
+        JsonNode sampleJson = new ObjectMapper().readTree(sampleValue);
+        String action = isSampleExists(getAlias(sampleJson)) ? "MODIFY" : "ADD";
+        String sampleSubmission = generateSampleSubmissionJson(action, sampleJson);
+        if (action.equals("ADD") || (action.equals("MODIFY") && parameters.isSampleUpdate())) {
+          getSubmitService().doJsonSubmission(sampleSubmission);
+        }
+        sampleValue = getAlias(sampleJson);
+      }
+      
+      Sample sample = getSample(sampleValue);
       fieldValue.setValue(sample.getBioSampleId());
       callback.notify(sample);
 
-    } catch (WebinCliException e) {
+    } catch (WebinCliException | JsonProcessingException e) {
       result.add(ValidationMessage.error(e));
     }
   }
+  
+  private Sample getSample(String sampleIdOrAlias){
+  
+    return  ExceptionUtils.executeWithRestExceptionHandling(() -> getSampleService().getSample(sampleIdOrAlias),
+            WebinCliMessage.SERVICE_AUTHENTICATION_ERROR.format(SampleService.SERVICE_NAME),
+            WebinCliMessage.SAMPLE_SERVICE_VALIDATION_ERROR.format(sampleIdOrAlias),
+            WebinCliMessage.SAMPLE_SERVICE_SYSTEM_ERROR.format(sampleIdOrAlias));
+  
+  }
+
+  private boolean isSampleExists(String sampleIdOrAlias){
+    try {
+      return getSampleService().getSample(sampleIdOrAlias)!=null;
+    }catch (HttpClientErrorException.NotFound notFound){
+      return false;
+    }catch (Exception ex){
+      throw WebinCliException.systemError(WebinCliMessage.SAMPLE_SERVICE_SYSTEM_ERROR.format(sampleIdOrAlias));
+    }
+  }
+  
+  private SampleService getSampleService(){
+    return 
+            new SampleService.Builder()
+                    .setWebinRestV1Uri(RemoteServiceUrlHelper.getWebinRestV1Url(parameters.isTest()))
+                    .setCredentials(parameters.getWebinServiceUserName(), parameters.getPassword())
+                    .setWebinAuthUri(RemoteServiceUrlHelper.getWebinAuthUrl(parameters.isTest()))
+                    .setBiosamplesUri(RemoteServiceUrlHelper.getBiosamplesUrl(parameters.isTest()))
+                    .setBiosamplesWebinUserName(parameters.getWebinServiceUserName())
+                    .setBiosamplesWebinPassword(parameters.getPassword())
+                    .build();
+  }
+  
+  private SubmitService getSubmitService(){
+    
+    return  new SubmitService.Builder()
+            .setWebinRestV2Uri(RemoteServiceUrlHelper.getWebinRestV2Url(parameters.isTest()))
+            .setUserName(parameters.getWebinServiceUserName())
+            .setPassword(parameters.getPassword())
+            .build();
+    
+  }
+  
+  
+  private boolean isJsonSample(String value){
+    return value.startsWith("{");
+  }
+  
+  private static String getAlias(JsonNode sampleJson) {
+    return sampleJson.get("alias").asText();
+  }
+
+  public String generateSampleSubmissionJson(String action, JsonNode sampleJson) {
```

Comment:
- createJsonSampleSubmission
- use the serializers from the webin-xml project if possible